src/category_theory/is_connected.lean

```diff
@@ -51,7 +51,7 @@ namespace category_theory
 A possibly empty category for which every functor to a discrete category is constant.
 -/
 class is_preconnected (J : Type v₂) [category.{v₁} J] : Prop :=
-(iso_constant : Π {α : Type v₂} (F : J ⥤ discrete α) (j : J),
+(iso_constant : Π {α : Type v₃} (F : J ⥤ discrete α) (j : J),
```

Comment:
Ooh, this is not what I meant... I think the definition doesn't need the extra universe... (Technically, if all functors to `discrete bool` are constant, then you should be able to deduce it for all other `\alpha` in all other universes, right?)

Reply:
I agree, I think I originally had the extra universe and it gave no benefit and made me write extra universe annotations

Reply:
Okay, I've reverted that change. I'm not excited to do this here.